modules/deepspace/assets/ships/alienCapital/alienCapital.json

```diff
@@ -0,0 +1,138 @@
+{
+  "size": 3,
+  "maxLife": 800,
+  "type": "big",
+  "price": 20000,
+  "e1Pos": "0.08 0.37",
+  "e2Pos": "0.08 0.63",
+  "displayName": "Alien Capital",
+  "engine": "core:imperialBigEngine",
+  "gunSlots": [
+    {
+      "position": "0.276 0.2935",
+      "isUnderneathHull": false,
+      "allowsRotation": true
+    },
+    {
+      "position": "0.276 0.7075",
+      "isUnderneathHull": false,
+      "allowsRotation": true
+    }
+  ],
```

Comment:
Same as above - 4 spaces. You should be able to format them en-masse in any decent editor.